Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -1391,6 +1391,8 @@ def rawToDict(raw):
             key_val_arr = raw.split(",")
             for key_val in key_val_arr:
                 single_key_val = key_val.split(":", 1)
+                if len(single_key_val) <= 1:
+                    single_key_val = key_val.split("=", 1)
                 if len(single_key_val) > 1:
```

Comment:
```suggestion
               else:
```

Reply:
This is not a valid change - notice that in the `if` we change the value of `single_key_val`